vendor.conf

```diff
@@ -21,7 +21,7 @@ github.com/golang/protobuf              925541529c1fa6821df4e44ce2723319eb2be768
 github.com/cyphar/filepath-securejoin   a261ee33d7a517f054effbf451841abaafe3e0fd # v0.2.2
 github.com/docker/go-units              47565b4f722fb6ceae66b95f853feed578a4a51c # v0.3.3
 github.com/urfave/cli                   cfb38830724cc34fedffe9a2a29fb54fa9169cd1 # v1.20.0
-golang.org/x/sys                        9eafafc0a87e0fd0aeeba439a4573537970c44c7 https://github.com/golang/sys
+golang.org/x/sys                        52ab431487773bc9dd1b0766228b1cf3944126bf https://github.com/golang/sys
```

Comment:
This matches the updated version in opencontainers/selinux; https://github.com/opencontainers/selinux/blob/v1.3.1/go.mod#L7